Jenkinsfile.github

```diff
@@ -0,0 +1,59 @@
+/*
+ *
+ *  MIT License
+ *
+ *  (C) Copyright 2023 Hewlett Packard Enterprise Development LP
+ *
+ *  Permission is hereby granted, free of charge, to any person obtaining a
+ *  copy of this software and associated documentation files (the "Software"),
+ *  to deal in the Software without restriction, including without limitation
+ *  the rights to use, copy, modify, merge, publish, distribute, sublicense,
+ *  and/or sell copies of the Software, and to permit persons to whom the
+ *  Software is furnished to do so, subject to the following conditions:
+ *
+ *  The above copyright notice and this permission notice shall be included
+ *  in all copies or substantial portions of the Software.
+ *
+ *  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ *  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ *  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+ *  THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+ *  OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+ *  ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+ *  OTHER DEALINGS IN THE SOFTWARE.
+ *
+ */
+@Library('csm-shared-library') _
+
+pipeline {
+
+    agent {
+        label "metal-gcp-builder"
+    }
+
+    options {
+        buildDiscarder(logRotator(numToKeepStr: "10"))
+    }
+
+    stages {
+        stage("Make RPM") {
+            steps {
+                sh "make rpm"
+            }
+        }
+
+        stage("Publish") {
+            steps {
+                script {
+                    publishCsmRpms(
+                        component: "sbps-marshal",
+                        pattern: "dist/rpmbuild/RPMS/noarch/*.rpm",
+                        os: "noos", 
+                        arch: "noarch",
+                        isStable: env.TAG_NAME ==~ /v\d+\.\d+\.\d+/
```

Comment:
Package is getting signed and placed into https://artifactory.algol60.net/artifactory/csm-rpms/hpe/stable, if it is invoked on a tag shaped as `vX.Y.Z`. Otherwise, package is not signed and placed into `/unstable`.